libsg3/ses/ses.pyx

```diff
@@ -329,10 +329,17 @@ cdef class EnclosureDevice(object):
         }
         return element_dict[index]
 
-    def decode_response(self, buff):
+    cdef str decode_response(self, char * buff):
+        cdef int i = 0;
+        with nogil:
+            while buff[i] != 0:
+                if buff[i] < 0 or buff[i] > 127:
+                    buff[i] = 26    # SUBstitute character
+                i += 1;
```

Comment:
Do we need semicolons here and above?

Reply:
Thanks, it was by mistake. Semicolons are removed.